### PR TITLE
discount_factor function

### LINKED ISSUE #65: Discount rate function
Write a function that returns the discounted cost for each cost

### PR DESCRIPTION
**Pull request by babakkhavari**
Function calcuating discount factor and years of analysis. These are both used for all costs and benefits. This fixes #65

### CODE REVIEW

File: onsstove/technology.py [hunk 1]
```diff
@@ -94,6 +94,25 @@ def paf(rr, sfu):
...
+        :return: discount factor to be used for all costs in the net benefit fucntion and the years of analysis
+        '''
+        if specs_file["Start_year"] == specs_file["End_year"]:
+            proj_life = 1
+        else:
+            proj_life = specs_file["End_year"] - specs_file["Start_year"]
+
+        year = np.arange(proj_life)
+
+        discount_factor = (1 + discount_rate_tech) ** year
```
Comment (camiloramirezgo) [inline onsstove/technology.py:113]: were is `discount_rate_tech` defined? if it is an attribute from the class then it should use `self`

**Review by camiloramirezgo — CHANGES_REQUESTED** [review]

**Comment by camiloramirezgo** [comment]
Also please link the issues that will be closed by this PR

File: onsstove/technology.py (hunk 1 above)
Comment (babakkhavari) [reply onsstove/technology.py:113]: It is defined in the specs file

Comment (camiloramirezgo) [reply onsstove/technology.py:113]: a ok, then you can make it as ` specs_file["discount_rate_tech"]` or something similar

**Pull request merged**